src/Aspire.Dashboard/Components/Controls/UserProfile.razor

```diff
@@ -0,0 +1,33 @@
+﻿<AuthorizeView>
+    <Authorized>
+        @if (_showUserProfileMenu)
+        {
+            <div class="profile-menu-container">
+                <FluentProfileMenu Initials="@_initials"
+                                   EMail="@_username"
+                                   FullName="@_name"
+                                   ButtonSize="@ButtonSize"
+                                   FooterLink="">
+                    <HeaderTemplate>
+                        <FluentStack VerticalAlignment="VerticalAlignment.Center">
+                            <FluentLabel>OpenID Connect</FluentLabel>
```

Comment:
This is what I referenced in one of the TODOs in the description. We can:

1. Leave it as-is
2. Set it to something else constant
3. Set it to something else constant and localize it
4. Figure out some way to get an appropriate value from an IdP
5. Remove it altogether

Reply:
Another option is just to remove the text. I think it's pretty clear what the popup means in context:

![image](https://github.com/dotnet/aspire/assets/350947/0d8529d8-a249-48fd-98a0-79f5f95e61b7)